js/components/accessibilityControls.js

```diff
@@ -88,6 +88,30 @@ AccessibilityControls.prototype = {
   },
 
   /**
+   * Determines whether or not the controller is in a state that allows seeking the video.
+   * @private
+   * @return {Boolen} True if seeking is possible, false otherwise.
+   */
+  canSeek: function() {
+    var seekingEnabled = false;
+    switch (this.controller.state.screenToShow) {
+      case CONSTANTS.SCREEN.PLAYING_SCREEN:
+      case CONSTANTS.SCREEN.PAUSE_SCREEN:
+      case CONSTANTS.SCREEN.END_SCREEN:
```

Comment:
Does End Screen cover replay, discovery, etc?

Reply:
These are the only three screens that show a progress bar (end has the replay button and the progress bar). We don't allow seeking on discovery, share or any of the other modals.

Reply:
Got it.